sky/templates/sky-serve-controller.yaml.j2

```diff
@@ -30,6 +30,11 @@ file_mounts:
   {{remote_catalog_path}}: {{local_catalog_path}}
   {%- endfor %}
 
+{%- if vpn_config is not none %}
+vpn:
+  {{vpn_config}}
+{%- endif %}
```

Comment:
lets add a todo/issue to implement a toggle for whether to open the controller to the public internet or not.